docs/vgclog_examples.md

```diff
@@ -706,3 +707,895 @@ Analyzing the structure and elements of this example log output shows that this
+
+<exclusive-end/>                          (STW pause ends)    
+
+<exclusive-start/>                        (STW pause starts)
+
+<cycle-start type="partial gc" />         (partial cycle starts)
+
+...
+
+<gc-op type="sweep" />                    (Sweep operation associated with global mark cycle runs)
+
+...
+
+<cycle-end type="partial gc"/>            (partial GC cycle ends)
+
+<exclusive-end/>                          (STw pause ends)
+
+```
+
+
+#### Global mark phase
+
+The first activity of the global mark cycle is a STW pause, recorded by an `<exclusive-start>` element. The GC pauses application threads to execute the cycle's initial operations.
+
+```xml
+<exclusive-start id="1152" timestamp="2021-02-26T11:17:25.033" intervalms="1931.263">
+  <response-info timems="3.082" idlems="1.041" threads="2" lastid="00000000006EDE00" lastname="RunDataWriter.1" />
+</exclusive-start>
+```
+
+The cycle is triggered when the allocation taxation threshold is reached. The <allocation-taxation> element records the memory threshold value that was in use when this example global mark cycle was triggered. The`taxation-threshold` attribute shows the taxation threshold is 1024MB (1,073,741,824), which is half the total memory of the *eden* space (2048 MB), as expected for threshold triggers of global mark cycles and increments. For more information taxation thresholds for the `balanced` policy, see [`balanced` examples](vgclog_examples.md#balanced-examples).
+
+```xml
+<allocation-taxation id="1153" taxation-threshold="1073741824" timestamp="2021-02-26T11:17:25.034" intervalms="1931.251" />
+```
+
+Details about the start of the global mark GC cycle are recorded by the `<cycle-start>` element. The cycle is recorded as type `global mark phase` with an `id=1154`. Any subsequent elements associated with this cycle have a `contextid=1154` to match the global mark GC cycle's `id`. You can use the `contextid` value to distinguish increments and operations of the global mark GC cycle from the partial cycles that interleave with it.
+
+```xml
+<cycle-start id="1154" type="global mark phase" contextid="0" timestamp="2021-02-26T11:17:25.034" intervalms="374365.075" />
+```
+
+The cycle begins with the STW subincrement of a global *mark* phase increment. The STW subincrement is recorded using the `<gc-start>` element of type `global mark phase`.
+
+```xml
+<gc-start id="1155" type="global mark phase" contextid="1154" timestamp="2021-02-26T11:17:25.034">
+  <mem-info id="1156" free="1442840576" total="4294967296" percent="33">
+    <remembered-set count="2197888" freebytes="162912768" totalbytes="171704320" percent="94" regionsoverflowed="3" regionsstable="130" regionsrebuilding="0"/>
+  </mem-info>
+</gc-start>
+
+```
+
+The `<gc-start>` element provides a snapshot of the free memory available in the heap and the status of marked objects. At the start of the increment, the heap is 33% free, with 1376 MB (1442840576 B) of the total 4096 MB (4294967296 B) heap free.
```

Comment:
this does not report anything about status  of marked object.
later gc-ops (mark increment) will do it

Reply:
No idea why I wrote this! 

'status of the remembered set'.